CoEpi/ui/onboarding/AttributedString.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+typealias FontAttributes = [NSAttributedString.Key: Any]
+
+struct AttributableString {
+    
+    let baseFont: UIFont
+    var storage = NSMutableAttributedString()
+    
+    let boldAttributes: FontAttributes
+    let normalAttributes: FontAttributes
+
+    init(baseFont: UIFont) {
+        
+        self.baseFont = baseFont
+        
+        let pointSize = baseFont.pointSize
+        let fontDescriptor = baseFont.fontDescriptor.withSymbolicTraits(.traitBold)
+            ?? UIFont.boldSystemFont(ofSize: pointSize).fontDescriptor
+        
+        let boldFont = UIFont(descriptor: fontDescriptor, size: pointSize)
+        
+        self.boldAttributes = [
+            .font: boldFont
+        ]
+        
+        self.normalAttributes = [
+            .font: baseFont
+        ]
+        
+    }
+    
+    func append(_ value: String, attributes: FontAttributes) {
+        let attributedString = NSAttributedString(string: "\(value) ", attributes: attributes)
+        storage.append(attributedString)
+    }
+    
+    func bold(_ value: String) -> AttributableString {
+        
+        append(value, attributes: boldAttributes)
+        
+        return self
+        
```

Comment:
I'd use blank lines only to separate block semantically. Seem unnecessary in small methods.